src/pages/FarmingPage.js

```diff
@@ -1,19 +1,43 @@
-import React, { useEffect } from "react";
+import React, { useState, useEffect } from "react";
 import "feather-icons";
 
 import { TYPE } from "../Theme";
 import Panel from "../components/Panel";
 import {
-  useLiqudityMiningCampaignData,
+  useLiquidityMiningCampaignData,
 } from "../contexts/PairData";
+
 import { PageWrapper, FullWrapper } from "../components";
 import { RowBetween } from "../components/Row";
 import Search from "../components/Search";
+import DropdownSelect from "../components/DropdownSelect";
 import { useMedia } from "react-use";
 import FarmingList from "../components/FarmingList";
 
 function FarmingPage() {
-  const campaignData = useLiqudityMiningCampaignData();
+
+  const activeCampaigns = useLiquidityMiningCampaignData("active");
+  const expiredCampaigns = useLiquidityMiningCampaignData("expired");
+  const [campaigns, setCampaigns] = useState({});
+  const [campaignStatus, setCampaignStatus] = useState("active");
```

Comment:
I think we should have and enum for active and inactive campaign and reuse it across the boars so its not hardcoded...

Reply:
also to merge it with options...